pkg/plugins/plugins.go

```diff
@@ -106,6 +118,86 @@ func (p PluginDTO) IncludedInSignature(file string) bool {
 	return true
 }
 
+func (p PluginDTO) Markdown(name string) []byte {
+	// nolint:gosec
+	// We can ignore the gosec G304 warning since we have cleaned the requested file path and subsequently
+	// use this with a prefix of the plugin's directory, which is set during plugin loading
+	path := filepath.Join(p.pluginDir, mdFilepath(strings.ToUpper(name)))
```

Comment:
You're wrongly assuming that the caller of this has cleaned the path. It might be true, but could also be false if something changes. I think we need to do the cleaning here as well. Possible to get rid of the nolint using what we've used for plugin dashboards?
https://github.com/grafana/grafana/blob/13d5ad2ce2a7283dc25dace1328cbf0c11fec639/pkg/plugins/manager/dashboards/filestore.go#L87-L94

Reply:
I forgot about that util func! Thanks for pointing that out.

In this case, I think `mdFilepath` should clean? But I will change this anyway to be clearer